lib/fog/openstack/requests/baremetal/list_drivers.rb

```diff
@@ -2,17 +2,18 @@ module Fog
   module Baremetal
     class OpenStack
       class Real
-        def list_drivers
+        def list_drivers(options = {})
           request(
```

Comment:
ah you are talking about options = {} unless options, I removed it from all places. It shouldn't be needed